includes/multi-currency/Compatibility/WooCommerceSubscriptions.php

```diff
@@ -142,7 +148,7 @@ public function maybe_disable_mixed_cart( $value ) {
 	 */
 	public function override_selected_currency( $return ) {
 		// If it's not false, return it.
-		if ( $return ) {
+		if ( $return || $this->running_filters ) {
```

Comment:
Do you think it'll make sense to be more specific about the `running_filters` variable name here? Maybe it'll be confusing in the future, and someone tries to use it in another method in this file, which may confuse the script?

Reply:
I thought about that, as well, but I couldn't think of a better name. What would you suggest? Maybe `currently_running`?

Reply:
No, I meant something like `$this->running_currency_override_filters` maybe. Because it'll only affect this method, and I guess it should be only affecting this method. Right?